index.js

```diff
@@ -32,7 +33,7 @@ CacheBreaker.prototype.cacheBreakPath = function(base, resource) {
   var extname = path.extname(resource);
   var basename = path.basename(resource, extname);
 
-  return path.join(dirname, basename + '.' + cs.substring(0, 10) + extname);
+  return url.resolve(dirname + '/', basename + '.' + cs.substring(0, 10) + extname);
```

Comment:
Please use `path.resolve` instead of `url`.   With `path.resolve` appending `/` shouldn't be necessary.

Let me know if this doesn't fix the underlying issue on windows and we can dive deeper!